pages/common/catimg.md

```diff
@@ -0,0 +1,21 @@
+# catimg
+
+> Image printing in the terminal.
+> See also: `pixterm` and `chafa`.
+> More information: <https://github.com/posva/catimg>.
+
+- Print a JPEG, PNG, or GIF to the terminal:
+
+`catimg {{path/to/file}}`
+
+- Use doubled [r]esolution:
+
+`catimg -r 2 {{path/to/file}}`
+
+- Disable 24-bit color for better [t]erminal support:
+
+`catimg -t {{path/to/file}}`
+
+- Specify custom [w]idth or [H]eight:
+
+`catimg {{-w|-H}} {{custom}} {{path/to/file}}`
```

Comment:
```suggestion
`catimg {{-w|-H}} {{1000}} {{path/to/file}}`
```

I don't know whether this is correct for what the command needs.

Reply:
What would improve the clarity here? Should I always set some sort of default value?

Reply:
Yes, we like to give an example parameter that can be given 

Like 

`sleep {{5}}`

is better than 

`sleep {{number}}`

Reply:
Alright! A reasonable value would be something like 40, so I'll set it to that.

```suggestion
`catimg {{-w|-H}} {{40}} {{path/to/file}}`
```

Reply:
Sure